src/test/java/com/marklogic/kafka/connect/RegexTest.java

```diff
@@ -0,0 +1,31 @@
+package com.marklogic.kafka.connect;
+
+import com.marklogic.kafka.connect.source.DslQueryHandler;
+import com.marklogic.kafka.connect.source.MarkLogicSourceConfig;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RegexTest {
```

Comment:
I think this name is a good indicator that those methods in the interface above should really be in an e.g. `QueryHandlerUtil` non-public class, and this then becomes `QueryHandlerUtilTest` which is a more self-documenting name than `RegexTest`, and should live in the "source" package as well.